js/vue/meetupEvents.vue

```diff
@@ -0,0 +1,96 @@
+
+<template>
+  <div>
+    <div class="meetups">
+      <div class="meetup-event" v-for="event in events" :key="event.id">
+        <h3 class="meetup-heading"> {{ event.name }}</h3>
```

Comment:
Rouge space?

Reply:
how dare it! Usually prettier catches my linting but it can't lint inside my vue scripts :(